README.md

```diff
@@ -5,22 +5,27 @@ pgmpy
 [![Code Health](https://landscape.io/github/pgmpy/pgmpy/dev/landscape.svg?style=flat)](https://landscape.io/github/pgmpy/pgmpy/dev)
 [![Join the chat at https://gitter.im/pgmpy/pgmpy](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/pgmpy/pgmpy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Python Library for Probabilistic Graphical Models  
-Documentation: [pgmpy](http://pgmpy.org/)  
-Mailing List: pgmpy@googlegroups.com  
+pgmpy is a python library for working with Probabilistic Graphical Models.
 
-Dependencies:
+Documentation  and list of algorithms we support is at our official site http://pgmpy.org/
+
+Our mailing list is at https://groups.google.com/forum/#!forum/pgmpy .
+
+We have our community chat at [gitter](https://gitter.im/pgmpy/pgmpy). We have a very welcoming and helpful community.
+
+Dependencies
 =============
-- Python 3.3
-- NetworkX 1.9.1
-- Scipy 0.12.1
-- Numpy 1.9.2
-- Cython 0.21
-- Pandas 0.15.1
+Pgmpy has following non optional dependencies
+- Python 2.7 or higher versions, Python 3 or higher versions
+- NetworkX 1.9.1 or higher
+- Scipy 0.12.1 or higher
+- Numpy 1.9.2 or higher
+- Cython 0.21 or higher
+- Pandas 0.15.1 or higher ( except 0.17.0 )
```

Comment:
@khalibartan Don't say higher, just the exact versions that we have. Might not work with all the versions.